src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+    static int taskCount = 0;
+    static int doneIndex = 0;
+
+    public static void echoCommandList(Task[] tasklist) {
+        System.out.println(horizontal + "Here are the tasks in your list:\n");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println("    " + (i+1) + ". [" + taskList[i].getTaskType() + "]" + "[" + taskList[i].getStatusIcon() + "] " + taskList[i].getDescription());
+        }
+        System.out.println(horizontal);
+    }
+
+    public static void echoCommandDone(String line) {
+        System.out.println(horizontal + "Nice! I\'ve marked this task as done:\n" + "   [" + taskList[doneIndex-1].getTaskType() + "][X] " + line + "\n" + horizontal);
+    }
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(introduction);
+
+        String line;
+        String[] splicedLine;
+        String commandWord;
+
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        String task;
+        String taskDate;
+
+        while (!line.equals("bye")) {
+
+            splicedLine = line.split(" ");
+            commandWord = splicedLine[0];
+
+            if (line.equals("list")) {
+                echoCommandList(taskList);
+            }
+            else if (commandWord.equals("done")) {
+                doneIndex = Integer.parseInt(splicedLine[1]);
+                echoCommandDone(taskList[doneIndex-1].getDescription());
+                taskList[doneIndex-1].markAsDone();
+            }
+            else if (commandWord.equals("todo")) {
+                task = line.substring(line.indexOf(' '));
+                taskList[taskCount] = new Todo(task);
+                System.out.println(horizontal + taskList[taskCount].toString() + "Now you have " + ++taskCount + " tasks on the list.\n" + horizontal);
+            }
+            else if (commandWord.equals("deadline")) {
+                task = line.substring(line.indexOf(' ')+1, line.indexOf('/')-1);
+                taskDate = line.substring(line.indexOf('/')+4);
+                taskList[taskCount] = new Deadline(task, taskDate);
+                System.out.println(horizontal + taskList[taskCount].toString() + "Now you have " + ++taskCount + " tasks on the list.\n" + horizontal);
+            }
+            else if (commandWord.equals("event")) {
+                task = line.substring(line.indexOf(' ')+1, line.indexOf('/')-1);
+                taskDate = line.substring(line.indexOf('/')+4);
+                taskList[taskCount] = new Event(task, taskDate);
+                System.out.println(horizontal + taskList[taskCount].toString() + "Now you have " + ++taskCount + " tasks on the list.\n" + horizontal);
```

Comment:
Line exceeds character limit of 110(hard limit 120). Please wrap the line with an indentation of 8 spaces (double indentation of parent line)